app/views/reports/new.html.erb

```diff
@@ -87,4 +87,26 @@
   </div>
 </div>
 
+<% if @edit %>
```

Comment:
better to move it to separate partial if you want to use it inside such condition